kong/init.lua

```diff
@@ -1278,6 +1278,10 @@ function Kong.balancer()
     -- record failure data
     local previous_try = tries[try_count - 1]
     previous_try.state, previous_try.code = get_last_failure()
+    if ngx.config.subsystem == "http" then
+      local peer_conn = require "resty.kong.peer_conn"
```

Comment:
Do we have any reason to require this library in runtime code path?

Reply:
require at the top of the file will cause it to fail when running it in the stream.